lazy_relay.js

```diff
@@ -523,6 +582,22 @@ function _observeCallReqContFrame(frame) {
 };
 
 function LazyRelayOutReq(conn, inreq) {
+    this.channel = null;
+    this.conn = null;
+    this.start = null;
+    this.remoteAddr = null;
+    this.logger = null;
+    this.inreq = null;
+    this.id = null;
+    this.serviceName = null;
+    this.callerName = null;
+    this.timeout = null;
+    this.operations = null;
+    this.timeHeapHandle = null;
+    this.alive = null;
```

Comment:
is having alive be essentially a "tri-state" intentional? I think it seems useful, but just wanted to check